controllers/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -171,6 +171,93 @@ func TestCreateUserAccountSuccessful(t *testing.T) {
 		})
 }
 
+func TestUserAccountSynchronizeSuccessfulWhenPropagatedClaimsModified(t *testing.T) {
```

Comment:
Hm... I would expect that we already have a test which tests MUR->UserAccount synchronization which we just need to update to include a PropogatedClaim check. Instead of creating a fully new test. Are you saying we don't have such a test?

Reply:
I don't think we have a general test.  There is `TestSyncMurStatusWithUserAccountStatuses` but it's specifically testing status synchronization.  There is also `TestCreateSynchronizeOrDeleteUserAccountFailed` which tests various failure scenarios.  If you're aware of any other tests that I may have missed please let me know.

Reply:
Hm... ok. Just wanted to avoid too much of code duplication.

Reply:
again, the same as for the `masteruserrecord.go` and `masteruserrecord_test.go`. 
you modified `sync.go` and there is `sync_test.go` file with unit tests covering the functions from the `sync.go` - they should be updated as well.